var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -9,7 +9,7 @@ class Netcdf(Package):
     homepage = "http://www.unidata.ucar.edu/software/netcdf/"
     url      = "ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-4.3.3.tar.gz"
 
-    version('4.4.0', 'f01cb26a0126dd9a6224e76472d25f6c')
+    version('4.4.0', 'cffda0cbd97fdb3a06e9274f7aef438e')
```

Comment:
While I was building ParaView, I got an error on netcdf deps complaining for a mismatch in MD5 hash: old hash as well as new one were printed, I just copy-pasted the new one in the package and restart the install with success.

I did not check the actual url used, as it is not set explicitely,  spack should compute it.
How we can check? You can build with the old hash?
Is there a better way to check for MD5 ?